src/style.scss

```diff
@@ -94,29 +94,28 @@ hr {
 }
 
 // Styling for dropdown links in the nav
-.dropdown-toggle {
-  color: #000 !important;
+.navbar-nav .nav-link.dropdown-toggle {
+  color: $black;
 }
 
 // Styling for the dropdown menu
 .dropdown-menu {
   text-align: center;
   background-color: $lightest-blue;
+
+  & .dropdown-item.active,
+  & .dropdown-item:active {
+    background-color: inherit;
+  }
 }
 
 // Styling for active navigation link
 a.active {
-  font-weight: 900 !important;
-  color: $teal !important;
+  font-weight: 900;
+  color: $teal;
```

Comment:
The color is still overridden when the important tag is taken away. It can be missed if you don't click on the page to get rid of the focus styling from `.borderLeftRight`. I'd suggest the following change: 

`.navbar .navbar-nav .nav-link.active {` to replace `a.active {`

in conjunction with my suggestion on line 218 regarding the `.borderLeftRight` selectors

Reply:
I found that it works as `.navbar-nav .nav-link.active` and we have a stylelint rule about max class chain being 2 - we can change the rule but for the time being that's what I updated it to.

Reply:
Rule updated and selector has the class added, thanks!